### PR TITLE
ci: Remove --progress-bar fallback for pip

### PR DESCRIPTION
**Pull request by seemethere**
All versions of pip that we currently support should have this flag so removing this should essentially be a no-op.

Also put the actual command into a variable so we only have to change it once next time instead of changing it in 3 places.

### CODE REVIEW

**Review by kit1980 — APPROVED** [review]

File: .ci/pytorch/common_utils.sh [hunk 1]
```diff
@@ -81,9 +81,10 @@ function pip_install_whl() {
 
 function pip_install() {
   # retry 3 times
-  # old versions of pip don't have the "--progress-bar" flag
-  pip install --progress-bar off "$@" || pip install --progress-bar off "$@" || pip install --progress-bar off "$@" ||\
-  pip install "$@" || pip install "$@" || pip install "$@"
+  pip_install_pkg="pip install --progress-bar off $@"
```
Comment (malfet) [inline .ci/pytorch/common_utils.sh:84]: pip3 please :)
```suggestion
  pip_install_pkg="pip3 install --progress-bar off $@"
```

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**